<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>251</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hello_Qt_OpenCV</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*
{
	font: 75 11pt;
	background-color: rgb(220, 220, 220);
}
QPushButton, QLineEdit, QGroupBox
{
	border: 2px solid rgb(0, 0, 0);
	border-radius: 10px;
	min-width: 80px;
	min-height: 35px;
}
QPushButton
{
	background-color: rgb(0, 255, 0);
}
QLineEdit
{
	background-color: rgb(0, 170, 255);
}
QPushButton:hover, QRadioButton:hover, QCheckBox:hover
{
	color: red;
}
QPushButton:!hover, QRadioButton:!hover, QCheckBox:!hover
{
	color: black;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="topHorizontalLayout">
      <item>
       <widget class="QLabel" name="inputLabel">
        <property name="text">
         <string>Input Image :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputLineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="inputPushButton">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/images/search.png</normaloff>:/images/search.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="filterTypeGroupBox">
      <property name="title">
       <string>Filter type</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QRadioButton" name="medianBlurRadioButton">
         <property name="text">
          <string>Median Blur</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="gaussianBlurRadioButton">
         <property name="text">
          <string>Gaussian Blur</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="bottomHorizontalLayout">
      <item>
       <widget class="QLabel" name="outputLabel">
        <property name="text">
         <string>Output Image :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="outputLineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="outputPushButton">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/images/search.png</normaloff>:/images/search.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCheckBox" name="displayImageCheckBox">
      <property name="text">
       <string>Display Image After Saving</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>625</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLanguage">
    <property name="title">
     <string>Language</string>
    </property>
    <addaction name="actionEnglish"/>
    <addaction name="actionFrench"/>
    <addaction name="actionGerman"/>
    <addaction name="actionTurkish"/>
   </widget>
   <addaction name="menuLanguage"/>
  </widget>
  <action name="actionFrench">
   <property name="text">
    <string>French</string>
   </property>
  </action>
  <action name="actionGerman">
   <property name="text">
    <string>German</string>
   </property>
  </action>
  <action name="actionTurkish">
   <property name="text">
    <string>Turkish</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="text">
    <string>English</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
